x-pack/legacy/plugins/lens/public/editor_frame_plugin/editor_frame/workspace_panel.tsx

```diff
@@ -147,8 +157,19 @@ export function WorkspacePanel({
   }
 
   return (
-    <DragDrop draggable={false} droppable={true} onDrop={onDrop}>
-      {renderVisualization()}
-    </DragDrop>
+    <EuiPageContent className="lnsPageContent">
+      <EuiPageContentHeader className="lnsPageContentHeader">
+        <EuiPageContentHeaderSection>
+          <EuiTitle size="xs">
+            <h2>New Visualization</h2>
```

Comment:
This text should be localized.

Reply:
I’m waiting for your save/load PR here, @chrisdavies As soon as that one is merged, we can show the actual name instead